core/src/rust_types.rs

```diff
@@ -485,10 +485,17 @@ impl RustEnumVariant {
 }
 
 /// Variant information shared among different variant types
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
 pub struct RustEnumVariantShared {
     /// The variant's ident
     pub id: Id,
     /// Comments applied to the variant
     pub comments: Vec<String>,
 }
+
+#[derive(Debug, PartialEq)]
+pub enum RustThing<'a> {
```

Comment:
We need an `#[allow(missing_docs)]` here. A brief description of why it needs to be duplicated would also be helpful.

Reply:
Also I'd prefer if we keep things like this `pub(crate)`, since this is still technically a hack.

Reply:
Alternatively, we could do what I mentioned in https://github.com/1Password/typeshare/pull/62#issuecomment-1426934009